articles/creating-your-first-google-web-story/index.md

```diff
@@ -0,0 +1,276 @@
+# Creating your first Google Web Story
+
+![Hero image](engineering-education/creating-your-first-google-web-story/hero.png)
+[Image Source: Unsplash](https://unsplash.com/photos/OU19pS58YY8)
+
+In October 2013, Snapchat introduced a new feature on its app called “Stories” which allowed users to create short consumable contents lasting only for 24 hours, and ever since then it gained popularity amongst Snapchat users and led to the adoption of the feature by other social apps like Instagram, Facebook, YouTube, WhatsApp, and Twitter.
+Introduced in 2018 as [AMP](https://amp.dev/) (Accelerated Mobile Pages) story, Google Web Story is owned by Google and powered by AMP technology. AMP (Accelerated Mobile Pages), developed by the AMP Open-Source team, is an open-source HTML platform designed for mobile web browsers to load web pages faster.
+
+### Table of Contents
+
+[Creating your first Google Web Story](#creating-your-first-google-web-story)
+[Table of Contents](#table-of-contents)
+[What is a Web Story?](#what-is-a-web-story)
+[Prerequisites](#prerequisites)
+[What differentiates a Web Story?](#what-differentiates-a-web-story)
+[Creating a Web Story](#creating-a-web-story)
+[Using NoCode](#using-nocode)
+[Using Code](#using-code)
+[How to make your Web Story visible on the Web](#how-to-make-your-web-story-visible-on-the-web)
+[Conclusion](#conclusion)
+[Further Reading](#further-reading)
+
+### What is a Web Story?
+
+> This is a web-based version of the popular “Stories” format that uses images, audio, videos, texts, and animations to create an engaging short consumable experience.
+
+The web prefix signifies the content is opened to everyone on the web and it is also short-lived or evergreen, depending on how the owner wants it to last.
+
+### Prerequisites
+
+To get started with Web Stories you’ll need some basic Knowledge of HTML, CSS, and JavaScript or no prior knowledge, a code editor to write the code, and a web browser to preview the code.
+
+### What differentiates a Web Story?
+
+Compared the conventional social app stories, Web stories are similar in terms of the overall consistent user experience of tapping back and forth to view previous and next stories. But some things that differentiate Web stories from social app stories include;
+
+**Control**
+Contrary to social app stories with Web stories, you're in control of how long the story should be on the Web and not having to worry that your beautifully crafted story would expire after 24 hours. The publisher host Web stories, which means the content, the copyright, and rules depend on the publisher's terms of choice.
```

Comment:
The publisher host`s` Web stories, which means ...

Reply:
Done